lib/vmdb/appliance.rb

```diff
@@ -20,6 +20,10 @@ def self.log_config_on_startup
       Vmdb::Appliance.log_diagnostics
     end
 
+    def self.PRODUCT_NAME
```

Comment:
why the UPPER_CASE naming for a method?

Reply:
Copying the convention https://github.com/ManageIQ/manageiq/blob/master/lib/vmdb/appliance.rb#L9-L15

I assume it's because they are effectively constants.